### DISCUSSION TITLE
Support balance other than ETH

### DISCUSSION THREAD

**Discussion by jcheese1** [rainbow-me/rainbowkit#336] [Ideas]
Please allow to display balance other than ETH, please :D

**Comment by nickbytes** [comment]
What would you like to see @jcheese1 ?

**Reply by jcheese1** [reply]
Hey! Would prefer if I can customize it to whatever balance I want to show. Im currently making an AMM and want to show our token balance (MAGIC) instead of ETH

**Reply by nickbytes** [reply]
🪄 nice 🪄 

You might find it easier for your use case to use the [Custom Connect Button](https://www.rainbowkit.com/docs/custom-connect-button). You'd be able to customize anything in your UI once your wallet is connected. So instead of showing the balance in ETH, you could show a user's MAGIC balance. You'd accomplish this by getting your `account` info, then reading the balance from MAGIC using one of [wagmi's hooks](https://wagmi.sh/docs/hooks/useContractRead).

**Reply by jcheese1** [reply]
Yep thats what I'm doing for now, but would be nice to utilize the UI of the current one without having to reimplement it! Not blocking at all, just a nice to have feature I think :D

**Reply by nickbytes** [reply]
Agreed. We're trying to think through ways this could work, and whether we want to expose our UI library as part of this package.
